packages/imt.sol/contracts/internal/InternalLeanIMT.sol

```diff
@@ -46,7 +46,7 @@ library InternalLeanIMT {
             revert LeafAlreadyExists();
         }
 
-        while (2 ** self.depth < self.size + 1) {
```

Comment:
maybe add a comment here to explain why checking just once is enough

Reply:
Sure, makes sense!

Reply:
@0xbok could you check if it's ok?